projects/plugins/crm/includes/ZeroBSCRM.Permissions.php

```diff
@@ -716,6 +718,18 @@ function zeroBSCRM_permsExport(){
 	    return false;
 	}
 
+/**
+ * Determine if the current user is allowed to manage segments.
+ *
+ * @return bool
+ */
+function jpcrm_perms_segments() {
+	$current_user = wp_get_current_user();
+	if ( $current_user->has_cap( 'admin_zerobs_customers' ) ) {
+		return true;
+	}
+	return false;
+}
```

Comment:
🪓

Reply:
Thank you - removed: f1d39a6d7b20cf8bbdff92a6c760ab3fafe579c9